pages/linux/procs.md

```diff
@@ -7,18 +7,18 @@
 
 `procs`
 
-- Show information about processes, if the commands which started them contain `zsh`:
+- List information about processes, if the commands which started them contain `zsh`:
 
 `procs {{zsh}}`
 
-- Show information about all processes sorted by CPU time in [a]scending or [d]escending order:
+- List information about all processes sorted by CPU time in ascending or descending order:
 
 `procs {{--sortd|--sorta}} cpu`
```

Comment:
```suggestion
`procs {{--sorta|--sortd}} cpu`
```
Shouldn't it be this? I am currently busy I will look into the review requests later today.

Reply:
Nice catch, yes, it should. 😄